### DISCUSSION TITLE
wagmi reacthook get balance return undefined when we build project.It not error when we run dev

### DISCUSSION THREAD

**Discussion by minhhoang2404** [wevm/wagmi#2427] [Q&A] [answered]
Error ContractFunctionExecutionError: An unknown error occurred while executing the contract function "balanceOf".

Raw Call Arguments:
  to:    0x438EC8Fb9583AeC39FdEe599d1086C5d885e6Cfb
  data:  0x70a08231000000000000000000000000d417e8da8254b4aa4fadd34ffff918f4889b2361
 
Contract Call:
  address:   0xd417E8Da8254b4Aa4faDd34FffF918F4889b2361
  function:  balanceOf(address account)
  args:               (0xd417E8Da8254b4Aa4faDd34FffF918F4889b2361)

Docs: https://viem.sh/docs/contract/readContract.html
Details: Cannot convert a BigInt value to a number
Version: viem@0.3.33
    at ge (getContractError.ts:55:10)
    at readContract.ts:100:11
    at h (regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (regeneratorRuntime.js:125:22)
    at Generator.throw (regeneratorRuntime.js:69:21)
    at r (asyncToGenerator.js:3:20)
    at l (asyncToGenerator.js:25:9)

**Comment by jxom** [comment, marked as answer]
https://github.com/hirosystems/stacks.js/issues/1096#issuecomment-946350299

**Reply by minhhoang2404** [reply]
thank you <3 <3 <3
